tests/lvm_installation.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { test, expect } from '@playwright/test';
```

Comment:
all are installations, so just `lvm`, and most likely will contain different lvm test in the future.

Reply:
updated.